libs/bluetooth/jswrap_bluetooth.h

```diff
@@ -33,6 +33,7 @@ typedef enum {
 #define BLETASK_IS_CENTRAL(x) ((x)>=BLETASK_CENTRAL_START && ((x)<=BLETASK_CENTRAL_END))
 
 extern JsVar *bleTaskInfo; // info related to the current task
+extern JsVar *blePromise; //defined here, used in jswrap_bluetooth.c and in ESP32 relevant bluetooth
```

Comment:
Do we need this? I'm not sure it is used and it'd be good to keep it private - `ble*Task` functions should hopefully provide everything that's needed to deal with the promises.

Reply:
I'm not too familar with the code here @jumjum123 did this.... Do you mean put in in a #ifdef ESP32 ?  Or ditch it all together?

Reply:
I have in mind, I had to do this for some reason.
But don't remember why :-(
Something like "ran into error during compiling"

Reply:
@jumjum123 
Looks like this can be removed as it's not getting used at this point.